api/pom.xml

```diff
@@ -398,7 +398,6 @@
                             <excludes>
                                 <exclude>**/edu/wgu/osmt/Application</exclude>
                                 <exclude>**/edu/wgu/osmt/ImportCommandRunner</exclude>
-                                <exclude>**/edu/wgu/osmt/security/SecurityConfig**</exclude>
```

Comment:
This line needs to be included. It has nothing to do with the build. It is about the osmt-api-lib artifact that Skills uses.

Reply:
Yes I know, but I just wanted to confirm something else since the main issue is about Security Configuration, that is why I specified in the commit message that it was for testing purposes..